src/components/PostsList/PostList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PostInfo from '../PostInfo/PostInfo';
+import { PreparedPost } from '../../types/PreparedPost';
+import './PostList.scss';
+
+type Props = {
+  posts: PreparedPost[],
+};
+
+const PostList: React.FC<Props> = ({ posts }) => (
```

Comment:
You can export the component right here  
```
export const PostList: React.FC<Props> = ({ posts }) => (...
```

end remove default export in the end